blog/2024-03-27-pants-2-20/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
+  authors: [cburroughs]
+  tags: [announcement]
+---
+
+# Pants 2.20.0 is released!
+
+<CaptionedImg src={require("./splash.jpeg").default}>
+  Photo by [Jongsun Leed](https://unsplash.com/@sarahleejs) /
+  [Unsplash](https://unsplash.com/photos/moon-eclipse-F-pSZO_jeE8)
+</CaptionedImg>
+
+{/* truncate */}
+
+We are pleased to announce Pants 2.20.0, the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.20.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/docs/getting-started).
+
+_Highlights in 2.20 include_:
+
+- ⚡ Support for the Ruff formatter, for both `BUILD` files and normal Python files.
+- 🔒 Built-in support for Terraform lockfiles.
+- 📁 Support for more remote-caching providers: file system and GitHub Actions Cache.
+- 🪥 New helpers for defining "adhoc" code-quality tools, without requiring a full plugin.
+- ☕ JVM third-party artifacts can now be read from `pom.xml` files, and other related improvements.
```

Comment:
What do you think about adding some more links here, similar to the later points?

```suggestion
- ⚡ Support for the Ruff formatter, for both `BUILD` files and normal Python files.
- 🔒 Built-in support for [Terraform lockfiles](https://www.pantsbuild.org/2.20/docs/terraform#lockfiles).
- 📁 Support for [more remote-caching providers](https://www.pantsbuild.org/2.20/docs/using-pants/remote-caching-and-execution/remote-caching): file system and GitHub Actions Cache.
- 🪥 New helpers for [defining "adhoc" code-quality tools](https://www.pantsbuild.org/2.20/reference/targets/code_quality_tool), without requiring a full plugin.
- ☕ JVM third-party artifacts [can now be read from `pom.xml` files](https://www.pantsbuild.org/2.20/reference/targets/jvm_artifacts), and other related improvements.
```